docs/streams/upgrade-guide.html

```diff
@@ -76,8 +76,13 @@ <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API
 
     <!-- Placeholder KIP-213 -->
     <!-- Placeholder KIP-307 -->
-    <!-- Placeholder KIP-479 -->
+    <p>
+        Before the 2.4.0 version of Kafa Streams, users of the DSL could not name the state stores involved in a Stream-Stream join.  If users added DSL operations upstream of the 
```

Comment:
nit: `stream-stream` join (or else `KStream-KStream`) here and below

Reply:
ack